src/features/events/components/EventOverviewCard/index.tsx

```diff
@@ -42,16 +56,40 @@ const EventOverviewCard: FC<EventOverviewCardProps> = ({
   const [locationId, setLocationId] = useState(
     eventData?.location.id ?? undefined
   );
+
+  const [startDate, setStartDate] = useState<Dayjs | undefined>(
```

Comment:
![image](https://user-images.githubusercontent.com/36491300/232440066-d5862909-141e-4f32-bbf5-f293071ac604.png)

I´ve tried to leave it at Dayjs only but as `EventData ` can be `null` I got errors, so my proposal is type the `startDate `as `<Dayjs | null>`

Reply:
This component will not be shown if `eventData` is null. I understand that typescript thinks that it can be null, because it's coming from a future, which is typed as potentially null. But that's not a realistic typing. Use `ZUIFuture` to make sure that typescript understands that it is always true, or change the component structure such that you can do `if (!eventData) return null` before you start rendering (which will be problematic now because that would make hooks conditional with the current setup).